grapher/core/Grapher.tsx

```diff
@@ -1485,7 +1485,7 @@ export class Grapher
                 this.yScaleType !== ScaleType.log
             )
 
-        if (this.facetStrategy === FacetStrategy.metric) return false
+        if (this.facetStrategy !== FacetStrategy.none) return false
```

Comment:
Hm, this is tricky, because there are some charts (like [CO₂ by source](https://ourworldindata.org/grapher/co2-by-source?stackMode=relative&facet=entity&country=SWE~AUS~DEU~GBR)) where this is valid, but then others where it isn't. 

Or more generally: We can have relative stacked charts, as long as the stacking strategy is not the same as the facet strategy. 

But the issue is that, again, like in many other cases developing this, we need information to flow up from the chart to the facets.

And then there is the additional complication that "relative" means different things for different charts, e.g. LineChart: https://ourworldindata.org/grapher/fossil-fuel-consumption-by-type?stackMode=relative&facet=entity&country=SWE~AUS~DEU~GBR

(This one is actually broken in facet mode, see [what it actually needs to look like](https://ourworldindata.org/grapher/fossil-fuel-consumption-by-type?stackMode=relative&time=1997..latest&facet=none&uniformYAxis=0&country=SWE))

For LineCharts, we shouldn't force disable the setting, because the relative means `value at current time / value at start time`.

I'm just dumping my thoughts for now, not sure which way I'm leaning on this. I was initially thinking that we could favour the safer approach (just disable it and don't worry about edge cases) but in some cases that would be kind of broken.

Reply:
Ah, I forgot that line charts can be in relative mode, since there few cases of it. Thanks for spotting it! I'll look at it before shipping anything.

Reply:
I've constrained it to only StackedDiscreteBar now just to get something out, but actually even there what we care about is whether we have more than one metric or not. We'll be back here when we do multi-metric selection.